crates/hyperqueue/src/client/commands/job.rs

```diff
@@ -32,6 +32,22 @@ pub struct JobCancelOpts {
     pub selector_arg: SelectorArg,
 }
 
+#[derive(Parser)]
+pub struct JobCatOpts {
+    /// Select job(s) to display
+    pub job_selector_arg: SelectorArg,
```

Comment:
To keep it simple, let's support only a single job for now. Selecting task IDs across jobs would be quite weird anyway :)